### DISCUSSION TITLE
Import hot wallet via xpriv [700k sat bounty]

### DISCUSSION THREAD

**Discussion by nostitos** [btcpayserver/btcpayserver#5377] [Ideas / Feature Requests]
xpriv is a very common wallet export format.

Would be useful for electrum seeds that cannot be imported but where xpriv can be exported.

**Comment by dennisreimann** [comment]
I'm pretty sure I discussed this with @NicolasDorier before. It was simply a matter of us not getting to implement it iirc.

**Comment by nostitos** [comment]
I'd need this, so attached a 700k sat bounty. (~200$)

**Reply by bitcoinshopkeeper** [reply]
Maybe this could be a plugin? If for some reason BTCpay server is not implementing it.

**Comment by 9boy64** [comment]
I want 2figure all this out faster !?

**Comment by nostitos** [comment]
Now worth about 500$ .....
